src/main/java/org/usfirst/frc/team1758/utilities/Controller.java

```diff
@@ -6,9 +6,10 @@
 public class Controller
 {
 	private Joystick joystick;
+	private final double threshold = 0.1;
```

Comment:
We should make this changable and not static. Best way to do that is to have a parameter that has a default value in the Constructor method. Again I know I did this.